extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -224,41 +224,51 @@ public function getWikiVariables() {
 	 * @desc Returns UA dimensions
 	 */
 	public function getTrackingDimensions() {
-		global $wgDBname, $wgUser, $wgCityId, $wgLanguageCode;
+		global $wgDBname, $wgUser, $wgCityId, $wgLanguageCode, $wgTitle;
 
-		$title = $this->getTitleFromRequest();
-		$article = Article::newFromID( $title->getArticleId() );
+		$title = $wgTitle;
+		$titleExists = true;
+		$dimensions = [];
 
-		if ( $article instanceof Article && $title->isRedirect() ) {
-			$title = $this->handleRedirect( $title, $article, [] )[0];
+		// Exception is thrown when empty title is send
+		// In that case we don't want to set dimensions which depend on title
+		// Title parameter is empty for URLs like /main/edit, /d etc. (all pages outside /wiki/ space)
+		try {
+			$title = $this->getTitleFromRequest();
+		} catch (Exception $ex) {
+			$titleExists = false;
+		}
+
+		if ( !empty ( $titleExists ) ) {
```

Comment:
I don't think there is a need for `$titleExists` variable. Just move what's in this block into `try { ... }` block.